scripts/build_test.xml

```diff
@@ -97,7 +101,10 @@
 	</target>
 
 	<target name="stage_test_material" depends="-create_test_directory">
-		<copy todir="${ascii.test.dir}">
+		<condition property="copy-encoding" value="IBM-1047" else="UTF-8">
+			<isset property="isZOS"/>
+		</condition>
+		<copy todir="${ascii.test.dir}" outputencoding="${copy-encoding}">
```

Comment:
We need to copy more carefully: not all files will use the same encoding; I suggest we use the `cp` utility on z/OS which should copy files and any tags properly.